variables.tf

```
@@ -111,20 +111,56 @@ variable "app_scale_limit" {
   default     = 1
 }
 
-variable "application_stack" {
-  description = "The application stack to configure for this Function App."
-
-  type = object({
-    dotnet_version              = optional(string)
-    use_dotnet_isolated_runtime = optional(bool, false)
-    java_version                = optional(string)
-    node_version                = optional(string)
-    python_version              = optional(string)
-    powershell_core_version     = optional(string)
-    use_custom_runtime          = optional(bool)
-  })
-
-  default = null
+# variable "application_stack" {
+#   description = "The application stack to configure for this Function App."
+
+#   type = object({
+#     dotnet_version              = optional(string)
+#     use_dotnet_isolated_runtime = optional(bool, false)
+#     java_version                = optional(string)
+#     node_version                = optional(string)
+#     python_version              = optional(string)
+#     powershell_core_version     = optional(string)
+#     use_custom_runtime          = optional(bool)
+#   })
+
+#   default = null
+# }
+
+variable "application_stack_dotnet_version" {
+  description = "The version of .NET to use for this Fuction App."
+  type        = string
+  default     = null
+}
+
+variable "application_stack_use_dotnet_isolated_runtime" {
+  description = "Should the .NET process for this Function App use an isolated runtime?"
+  type        = bool
+  default     = false
+}
+
+variable "application_stack_java_version" {
+  description = "The version of Java to use for this Fuction App."
+  type        = string
+  default     = null
+}
+
+variable "application_stack_node_version" {
+  description = "The version of Node to use for this Fuction App."
```

Full name is usually displayed as `Node.js`:

```suggestion
  description = "The version of Node.js to use for this Fuction App."
```